docs/02.API-REFERENCE.md

```diff
@@ -3896,3 +3897,79 @@ jerry_parse_and_save_literals (const jerry_char_t *source_p,
 - [jerry_init](#jerry_init)
 - [jerry_cleanup](#jerry_cleanup)
 - [jerry_register_magic_strings](#jerry_register_magic_strings)
+
+
+# Miscellaneous functions
+
+## jerry_set_vm_exec_stop_callback
+
+**Summary**
+
+When JERRY_FEATURE_VM_EXEC_STOP is enabled a callback function can be
+specified by this function. This callback is periodically called when
+JerryScript executes an ECMASCript program.
+
+If the callback returns with undefined value the ECMAScript execution
+continues. Otherwise the result is thrown by the engine (if the error
+flag is not set for the returned value the engine automatically sets
+it). The callback function might be called again even if it thrown
+an error. In this case the function must throw the same error again.
+
+Frequently calling a callback in the main executor loop reduces
+its performance. The `frequency` argument allows decreasing
+this overhead. If its value is `N`, the callback is called only
+every `N` times. The minimum value of `N` is `1` (`0` is converted
```

Comment:
Please recommend me a good sentence then:

This is an extra layer for improving performance. Every time when the callback could be called, we decrease a counter, and only call the callback if it is 0. We also reset it back to frequency (that is why frequnce must be > 0, to avoid underflow). The dividers of CPU clock works this way. Basically if you set to to N then N-1 times the call is skipped, and only the Nth time it is executed.